R/print.data.table.R

```diff
@@ -15,6 +15,8 @@ print.data.table = function(x, topn=getOption("datatable.print.topn"),
   # trunc.cols - should only the columns be printed that can fit in the console? (FALSE)
   if (!col.names %chin% c("auto", "top", "none"))
     stop("Valid options for col.names are 'auto', 'top', and 'none'")
+  if (!is.logical(trunc.cols))
```

Comment:
Could check if NA as well

Reply:
Related: could use ```!isTRUEorFALSE(trunc.cols)```

Reply:
Now added check for NA. Hadn't ever used `isTRUEorFALSE()` so I didn't use that this time.